CMakeLists.txt

```diff
@@ -1238,7 +1238,7 @@ include_directories(${ROCKSDB_INCLUDE_DIR})
 
 set(ROCKSDB_LIBS rocksdb;${SNAPPY_LIB})
 
-add_dependencies(rocksdb snappy-static)
+add_dependencies(rocksdb snappy)
```

Comment:
afair we patched the snappy lib to have a distinguishable target for this?

Reply:
I don't know. I don't see what we need it for. At least that library target is not available in upstream Snappy.

Reply:
Discussed this in a side channel with @dothebart : it is possible that we need an extra target for this to build with Ninja. However, building with Ninja is currently unsupported and fails for various other reasons. Thus we will not make a potential adjustment for Ninja in the context of this PR, so we can get rid of one extra patch on top of Snappy.